src/aria/popups/Popup.js

```diff
@@ -712,12 +729,23 @@ module.exports = Aria.classDefinition({
                     modalMaskZIndex = this.computedStyle.zIndex;
                 }
 
-                this.modalMaskDomElement.style.cssText = ['left:0px;top:0px;', 'width:', containerSize.width, 'px;', 'height:',
-                        containerSize.height, 'px;', 'z-index:', modalMaskZIndex, ';', 'position:absolute;display:block;'].join('');
+                modalMaskDomElement.style.cssText = [
+                    'left:0px;',
+                    'top:0px;',
+                    'width:', containerSize.width, 'px;',
+                    'height:', containerSize.height, 'px;',
+                    'z-index:', modalMaskZIndex, ';',
+                    'position:absolute;',
+                    'display:block;'
+                ].join('');
+
+                if (conf.waiAria && conf.modal) {
```

Comment:
There is no need to check `conf.modal` here, because we are already in the `if (modalMaskDomElement) { ... }`.